.github/workflows/package-on-fork.yaml

```diff
@@ -0,0 +1,127 @@
+# THIS WORKFLOW WILL GET TRIGGERED WHEN PR IS MERGED IN MASTER
+name: Package On Fork PR Merge
+env:
+  APPINSIGHTS: "${{ vars.APPINSIGHTS }}"
+  DEFAULTPACKAGEVERSION: "${{ vars.DEFAULTPACKAGEVERSION }}"
+  BLOB_CONN_STRING: "${{ secrets.BLOB_CONN_STRING }}"
+  BRANCH_NAME: "${{ github.event.pull_request.head.ref }}"
+
+on:
+  pull_request_target: 
+    branches:
+      - master
+    paths:
+      - Solutions/**
+    types:
+      - closed
+
+permissions:
+  contents: write
+  pull-requests: write
+  packages: write
+
+jobs:
+  if_merged:
+    if: ${{ github.actor != 'dependabot[bot]' && github.event.pull_request.merged && github.event.pull_request.head.repo.fork }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
+        with:
+          fetch-depth: 0
+
+      - id: checkAutomatedPR
+        name: check-Automated-PR
+        env:
+          BranchName: ${{ github.event.pull_request.head.ref }}
+        shell: pwsh
+        run: |
+          $forkPRBranchName = "$env:BranchName"
+          Write-Host "Fork Branch Name is $forkPRBranchName"
+          $isAutomatedPR = $false
+          if ($forkPRBranchName -like '*automated-pr')
+          {
+            Write-Host "Skipping packaging as it is an automated fork pr!"
+            $isAutomatedPR = $true
+          }
+          Write-Output "isAutomatedPR=$isAutomatedPR" >> $env:GITHUB_OUTPUT
+          Write-Host "Is this Pull Request autogenerated $isAutomatedPR"
+
+      - id: validateAndCreatePackage
+        name: validate-and-create-package
+        shell: pwsh
+        if: ${{ success() && steps.checkAutomatedPR.outputs.isAutomatedPR }}
```

Comment:
It shouldn't be automated PR check. verify

Reply:
done